### PR TITLE
add user roles and display role in maintenance page

### PR DESCRIPTION
**Pull request by kculala**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -16,4 +16,6 @@ application/logs/*
 !application/logs/.htaccess
 /vendor/
 /nbproject/private/
-/nbproject/
\ No newline at end of file
+/nbproject/
+
+tmp/*
```
Comment (achlin) [inline .gitignore:21]: Is this needed now with the workaround?

File: application/views/_menubar.php [hunk 2]
```diff
@@ -7,4 +7,11 @@
     {menudata}
     <li><a href="{link}">{name}</a></li>
     {/menudata}
+    <li class="dropdown">
+      <a class="dropdown-toggle" data-toggle="dropdown" href="#">User Role<b class="caret"></b></a>
```
Comment (achlin) [inline application/views/_menubar.php]: Adjust indent.

**Review by achlin — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (kculala) [reply .gitignore:21]: good point. removed redundant line

File: application/views/_menubar.php (hunk 2 above)
Comment (kculala) [reply application/views/_menubar.php]: done

File: .gitignore (hunk 1 above)
Comment (kculala) [reply .gitignore:21]: left as is per discussion offline about tmp folder

**Review by achlin — APPROVED** [review]

**Pull request merged**